File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec-vnoreny.js
// Deklarace a inicializace
const vel = 4; // Inicializace velikosti ctverce

// Vnejsi cyklus pro radky
for(var i = 1; i <= vel; i++) {

    // Inicalizujeme promennou pro vypsani radky
    var rad = "";

    // Vnitrni cyklus pro jednotlive hvezdicky
    for(var j = 1; j <= vel; j++) {            

        // Pridame do textoveho retezce hvezdicku
        rad += "* ";

    }

    // Vypiseme radku
    console.log(rad);

}

File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec-vnoreny.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace
    const int vel = 4; // Inicializace velikosti ctverce

    // Vnejsi cyklus pro radky
    for(int i = 1; i <= vel; i++) {

        // Vnitrni cyklus pro jednotlive hvezdicky
        for(int j = 1; j <= vel; j++) {            

            // Vypiseme hvezdicky
            cout << "* ";

        }

        // Vypiseme novou radku
        cout << endl;

    }

}
